source/_patterns/01-elements/link/link.scss

```diff
@@ -26,42 +27,40 @@
 			color: initial;
 		}
 	}
+	&[data-variant="internal"] {
+		@include icon(glyph(link), 20, "outline", "after");
+	}
+	&[data-variant="external"] {
+		@include icon(glyph(link-external), 20, "outline", "after");
+	}
 
+	&[data-variant="internal"],
+	&[data-variant="external"] {
+		color: #282d37;
+		&::after {
+			margin-top: -3px;
+			--icon-margin-before: #{to-rem($pxValue: 4)};
+		}
+		&:focus-visible {
+			outline-offset: -1px;
+			padding: 0 2px 0 2px;
+		}
+	}
 	// Sizes
 	&%size-Small {
 		&:not(.is-icon-text-replace) {
-			font-size: to-rem($pxValue: 14);
-		}
-		&[rel],
-		&[data-icon],
-		&[data-icon-before] {
-			@include iconMeta(20);
```

Comment:
to me it seems like we still need to include smaller icons here – as a dirty hack it would be fine to define `@include iconMeta(20);` here with an additional `font-size: 16px`, that we could remove as soon as those icons have been added to the icon font.